scripts/obtain/dotnet-install.sh

```diff
@@ -115,6 +115,24 @@ get_current_os_name() {
     if [ "$uname" = "Darwin" ]; then
         echo "osx"
         return 0
+    else
+        if [ "$uname" = "Linux" ]; then
+            echo "linux"
+            return 0
+        fi
+    fi
+    
+    say_err "OS name could not be detected: $ID.$VERSION_ID"
+    return 1
+}
+
+get_distro_specific_os_name() {
```

Comment:
Also, as of a few hours ago, on RHEL, the first download link fails. The alt one works.